using-mobile-extensions/adobe-edge-identity/edge-identity-api-reference.md

```diff
@@ -0,0 +1,520 @@
+# Edge Identity API Reference
+
+## getExperienceCloudId
+
+{% tabs %}
+{% tab title="Android" %}
+### getExperienceCloudId
+
+This API retrieves the ECID that was generated when the app was initially launched.
+
+This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. The values are returned via the [AdobeCallback](../mobile-core/mobile-core-api-reference.md#public-classes).
+
+When [AdobeCallbackWithError](../mobile-core/mobile-core-api-reference.md#public-classes) is provided, and you are fetching the ECID from the Mobile SDK, the timeout value is 500ms. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate [AdobeError](../mobile-core/mobile-core-api-reference.md#public-classes).
+
+**Java**
+
+**Syntax**
+
+```java
+public static void getExperienceCloudId(final AdobeCallback<String> callback);
+```
+
+* _callback_ is invoked after the ECID is available.
```

Comment:
Should we specify callbacks can be called on different threads? I think we do in our other docs.